app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -38,14 +38,15 @@ tableEmptyText=No table data.
 notificationEmptyText=Top publisher visits
 syncEmptyText=Sync settings coming soon.
 bitcoin=Bitcoin
-bitcoinAdd=Use an existing Bitcoin wallet
+bitcoinAdd=Use your existing bitcoin wallet/account
+bitcoinAddDescription=Use any BTC wallet that can transfer bitcoin to your Brave wallet.
 bitcoinBuy=Buy Bitcoin
 viewPaymentHistory=View Payment History
 paymentHistoryTitle=Your Payment History
 paymentHistoryFooterText=Your next payment submission is {{reconcileDate}}.
 paymentHistoryOKText=OK
 bitcoinAddress=Your Brave wallet address is:
-bitcoinPaymentURL=Scan the QR code or transfer {{amount}} to:
+bitcoinPaymentURL=Your Brave wallet address
```

Comment:
this is no longer the payment URL, so it should be merged into `bitcoinAddress` above